library/Ivoz/Provider/Domain/Service/RoutingPattern/UpdateByBrand.php

```diff
@@ -49,10 +49,10 @@ public function execute(BrandInterface $brand)
              */
             $routingPatternDto = RoutingPattern::createDto();
             $routingPatternDto
-                ->setNameEs($country->getName()->getEs())
-                ->setNameEn($country->getName()->getEn())
-                ->setNameCa($country->getName()->getCa())
-                ->setNameIt($country->getName()->getIt())
+                ->setNameEs($country->getName()->getEs() ?? '')
+                ->setNameEn($country->getName()->getEn() ?? '')
+                ->setNameCa($country->getName()->getCa() ?? '')
+                ->setNameIt($country->getName()->getIt() ?? '')
```

Comment:
We should create another issue for fixing this: Country name translations should not be nullable. We populate this readonly table.